docs/source/usage.rst

```diff
@@ -3,108 +3,81 @@
-
-Preparing your repository
--------------------------
-
-``repo2docker`` looks for configuration files in the repository being built
-to determine how to build it. It is philosophically similar to
-`Heroku Build Packs <https://devcenter.heroku.com/articles/buildpacks>`_.
-
-In general, ``repo2docker`` uses the same configuration files as other software
-installation tools, rather than creating new custom configuration files.
-These files are described in :ref:`config-files`.
-
-``repo2docker`` configuration files are all **composable** - you can use any number
-of them in the same repository.
-
-Locating and composing configuration files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. builds a Docker image from a git repo
+2. runs a Jupyter server within the image to explore the repo
 
-``repo2docker`` will look for configuration files located in two places:
+To ensure you can run the software in your repository, you must
 
-* A folder named ``binder`` in the root of the repository.
-* The root of the repository.
+repo2docker is called with a simple command::
 
-There are a few notable rules for composition precedence and build priority:
+  jupyter-repo2docker <URL-or-path to repo>
 
-* If the folder ``binder`` is located at the top level of the repository,
-  **only configuration files in the** ``binder`` **folder will be considered**.
-* If a Dockerfile is present, **all other files will be ignored**.
-* ``environment.yml`` **takes precedent over**
-  ``requirements.txt``. If you wish to install ``pip`` packages
-  with ``environment.yml``, please use the
-  ``pip:`` key as described in the `conda documentation`_.
+where ``<URL-or-path to repo>`` is a URL or path to the source repository.
 
-For a list of repositories demonstrating various configurations, see
-`binder examples <https://github.com/binder-examples>`_.
+For example, use the following to build an image of the
+`Python Data Scinece Handbook <https://github.com/jakevdp/PythonDataScienceHandbook>`_::
 
-Preparing a repo to build JupyterHub-ready images
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  jupyter-repo2docker https://github.com/jakevdp/PythonDataScienceHandbook
 
-It is possible to use ``repo2docker`` to build JupyterHub-ready
-Docker images. For this to work properly, the version of the ``jupyterhub``
-package in your git repository must match the version in your JupyterHub
-deployment. For example, if your JupyterHub deployment runs ``jupyterhub==0.8``,
-you should put the following in ``requirements.txt`` or ``environment.yml``::
+To build a particular branch and commit, we use the argument ``--ref`` to
+specify the ``branch-name/commit-hash``::
 
-  jupyterhub==0.8.*
+  jupyter-repo2docker https://github.com/jakevdp/PythonDataScienceHandbook --ref master/599aa0fe3f882c0001670e676e5a8d43b92c35fc
 
-Running ``repo2docker`` locally
--------------------------------
+Building the image may take a few minutes.
```

Comment:
After this sentence.

..tip::

For reproducibility, such as for journal publications, we recommend specifying a commit-hash instead of using the master branch which will change over time.